pandas/core/nanops.py

```diff
@@ -633,10 +633,31 @@ def _kendall(a, b):
     def _spearman(a, b):
         return spearmanr(a, b)[0]
 
+    def _biweight_midcorrelation(a, b):
+        a_median = a.median()
+        b_median = b.median()
+
+        # Median absolute deviation
+        a_mad = (a - a_median).abs().median()
```

Comment:
It all ends up dispatched to call the same internal method.

On Thu, Apr 16, 2015 at 12:28 PM, Olga Botvinnik <notifications@github.com>
wrote:

>> @@ -633,10 +633,31 @@ def _kendall(a, b):
>>      def _spearman(a, b):
>>          return spearmanr(a, b)[0]
>>  
>> +    def _biweight_midcorrelation(a, b):
>> +        a_median = a.median()
>> +        b_median = b.median()
>> +
>> +        # Median absolute deviation
>> +        a_mad = (a - a_median).abs().median()
> this is abs of the series, won't `series.abs()` be faster than using the python builtin `abs`? that was my thinking behind the code
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/pydata/pandas/pull/9826/files#r28541564